src/exm-backtrace.c

```diff
@@ -54,18 +60,25 @@ exm_backtrace_full_cb (GString    *string_builder,
 void
 exm_backtrace_init (char *filename)
 {
-#ifdef BACKTRACE_SUPPORTED
+#if WITH_BACKTRACE
+
+# ifdef BACKTRACE_SUPPORTED
     state = backtrace_create_state (filename, 0,
                                     exm_backtrace_error_cb,
                                     NULL);
-#else
+# else
     g_warning ("Backtraces are not supported.\n");
+# endif
+
+#else
+    g_warning ("Backtraces were not enabled at build time.\n");
```

Comment:
```suggestion
    g_info ("Backtraces were not enabled at build time.\n");
```

This shouldn't be a warning as it's a built-time flag

Reply:
Sure, change applied.